### ISSUE TITLE
Settings failed to load error in the dev tools console

### ISSUE THREAD

**Issue by jtpio** [jupyter/notebook#7062]
Currently seeing this error on the dev tools console on the latest `main`:

![image](https://github.com/jupyter/notebook/assets/591645/bf4fb57e-31e9-4486-87b6-b65a6d5954dc)

```
Settings failed to load for (@jupyterlab/metadataform-extension:metadataforms) 
```

And

```
Settings failed to load for (@jupyterlab/cell-toolbar-extension:plugin)
```

**Comment by jtpio** [comment]
cc @brichet who might know more about this and whether this is related to a change upstream in JupyterLab

**Comment by jtpio** [comment]
The messages seem to originate from the following:

https://github.com/jupyter/notebook/blob/2b4b422f0f044994960803c48a7c3a9dc9abdb0a/packages/tree-extension/src/index.ts#L211-L227

**Comment by brichet** [comment]
I can't find any related changes in Jupyterlab.
Do you know if this warning was not present in previous versions ?

The `settingRegistry.load` function seems to fail when the plugin is not activated.
For me it is expected, this message occurs on the tree view only, when the Notebook plugins are not loaded (so not the metadataform and cell toolbar either).
But these plugins should probably not be in the `pluginSet` variable.

**Comment by jtpio** [comment]
> For me it is expected, the Notebook plugins are not loaded in the tree view (so not the metadataform and cell toolbar either).

This logic was added so the plugins not activated on the tree page would still show up in the settings editor (which is only available on the tree page): https://github.com/jupyter/notebook/pull/6750

**Comment by brichet** [comment]
I don't think it can work for plugin with a transformation function, as these functions are registered on plugin activation. 
Both warnings concern plugins with a transformation function.
The only recent related PR I can see is https://github.com/jupyterlab/jupyterlab/pull/14990, but AFAIK it couldn't work before.

It seems that the message was already present with previous releases of Notebook: 
<img src=https://github.com/jupyter/notebook/assets/32258950/c33c400d-0dcb-4e09-a4ca-e24bc11f4321 width=400px>

**Comment by jtpio** [comment]
Thanks @brichet for the details :+1: 

> The only recent related PR I can see is [jupyterlab/jupyterlab#14990](https://github.com/jupyterlab/jupyterlab/pull/14990), but AFAIK it couldn't work before.

Yeah I saw a couple of PRs on JupyterLab that could have been related, but they are not backported to `4.0.x` so not used in Notebook 7 (for now).

**Comment by jtpio** [comment]
Maybe we should open the settings editor in one of the side panels instead of the file browser page (like the extension manager).

That way it would pick up the plugins loaded on the current page, and would not require having to load them explicitly.

**Comment by brichet** [comment]
This should work for loading settings, but the panel is quite large with the current display. It can be difficult to navigate if it's in a side panel.

**Comment by capey0** [comment]
In the latest version of Chrome devtools console:

I get 100s of the following:
"Skipping this shortcut because there are no actionable keys on this platform" issued from 5649.bundle.js

Then I get the same messages as OP. 

I am using:
Windows 11
Python 3.12.x
Used "pip3 install ..." to get Jupyter
Jupyter Dark theme
NO Anaconda

The result is the "increase/decrease font sizes" in the menus do not work. The color schemes for highlighting different data types and keywords also does not work.

**Comment by capey0** [comment]
This morning I opened up Jupyter Notebook and the keyword colorization seems to be working and so does the indenting. However, I now see this in the dev window when I try to change the font size:
![image](https://github.com/jupyter/notebook/assets/5092281/8b6d3d64-ac26-498a-9442-15625c9107ac)

**Comment by capey0** [comment]
Ok, frankly this sucks! The colorization has disappeared and so has the indenting.

Still getting all the above messages. 

:(!
